patches/api/0490-voidDamageThreshold-Height-at-which-entiies-take-voi.patch

```diff
@@ -0,0 +1,39 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Axionize <154778082+Axionize@users.noreply.github.com>
+Date: Wed, 25 Sep 2024 04:43:26 -0400
+Subject: [PATCH]   voidDamageThreshold - Height at which entiies take void
+ damage in the world   voidDamageAmount - Amount of void damage entities take
+ per tick in the void
+
+
+diff --git a/src/main/java/org/bukkit/World.java b/src/main/java/org/bukkit/World.java
+index d8a23aa0d898ca3360757721e38ddb97387f7d21..ec1ba8e4164da278f55441e1f3bb4edf724a45d3 100644
+--- a/src/main/java/org/bukkit/World.java
++++ b/src/main/java/org/bukkit/World.java
+@@ -53,6 +53,26 @@ import org.jetbrains.annotations.Nullable;
+ public interface World extends RegionAccessor, WorldInfo, PluginMessageRecipient, Metadatable, PersistentDataHolder, Keyed, net.kyori.adventure.audience.ForwardingAudience { // Paper
+ 
+     // Paper start
++    /**
++     * @return Sets the damage applied to the player per-tick when they are in the void in this world
```

Comment:
The method isn't returning anything, `@return` does not make sense here.
It is also missing a `@param voidDamageAmount` description.

Same for the threshold.